core/common/test/CommonBufferTest.kt

```diff
@@ -47,29 +47,26 @@ class CommonBufferTest {
     }
   }
 
-  /** Buffer's toString is the same as ByteString's.  */
-//  @Test fun bufferToString() {
-//    assertEquals("[size=0]", Buffer().toString())
-//    assertEquals(
-//      "[text=a\\r\\nb\\nc\\rd\\\\e]",
-//      Buffer().writeUtf8("a\r\nb\nc\rd\\e").toString()
-//    )
-//    assertEquals(
-//      "[text=Tyrannosaur]",
-//      Buffer().writeUtf8("Tyrannosaur").toString()
-//    )
-//    assertEquals(
-//      "[text=təˈranəˌsôr]",
-//      Buffer()
-//        .write("74c999cb8872616ec999cb8c73c3b472".decodeHex())
-//        .toString()
-//    )
-//    assertEquals(
-//      "[hex=0000000000000000000000000000000000000000000000000000000000000000000000000000" +
-//        "0000000000000000000000000000000000000000000000000000]",
-//      Buffer().write(ByteArray(64)).toString()
-//    )
-//  }
+  @Test fun bufferToString() {
```

Comment:
We could also add a write/read test for a surrogate pair like https://github.com/Kotlin/kotlinx.serialization/blob/ba7d17a15329840a85e823c6d4bdc3595d8a19c6/formats/json-tests/commonTest/src/kotlinx/serialization/features/EmojiTest.kt#L12

Reply:
I see, this test in `Utf8Test`